hyperspy/io_plugins/tiff.py

```diff
@@ -159,7 +161,11 @@ def file_reader(filename, record_by='image', force_read_resolution=False,
             dtype = np.dtype({'names': names[:lastshape],
                               'formats': [dtype] * lastshape})
             shape = shape[:-1]
-        op = {tag.name: tag.value for tag in tiff.pages[0].tags}
+        if LooseVersion(tifffile.__version__) >= LooseVersion("2020.2.16"):
```

Comment:
Oh, this. Are you sure that is version the behaviour changes of tiffiles?

Reply:
Based on the [GitHub releases page](https://github.com/cgohlke/tifffile/releases) and some quick testing, this check looks correct.